modules/mono/editor/godotsharp_editor.cpp

```diff
@@ -278,13 +278,11 @@ GodotSharpEditor::GodotSharpEditor(EditorNode *p_editor) {
 		about_label->set_autowrap(true);
 		String about_text =
 				String("C# support in Godot Engine is a brand new feature and a work in progress.\n") +
-				"It is at the alpha stage and thus not suitable for use in production.\n\n" +
+				"It is currently in an alpha stage and is not suitable for use in production.\n\n" +
 				"As of Godot 3.0, C# support is not feature-complete and may crash in some situations. " +
```

Comment:
This and the next line also need to be updated.

Reply:
Done. I only kept the "3.x" on the next line, as "3.1.x" is kinda implied by "3.x", and even if it isn't, the majority of C# changes will be introduced in the "3.x.0" releases.